broker/src/main/java/io/zeebe/broker/clustering/base/topology/TopologyManagerImpl.java

```diff
@@ -125,45 +112,42 @@ public void event(ClusterMembershipEvent clusterMembershipEvent) {
 
   // Remove a member from the topology
   private void onMemberRemoved(BrokerInfo brokerInfo) {
-    final NodeInfo nodeInfo = topology.getMember(brokerInfo.getNodeId());
-    if (nodeInfo != null) {
-      topology.removeMember(nodeInfo);
-      notifyMemberRemoved(nodeInfo);
-    }
+    brokerInfo.consumePartitions(
+        partition -> removeIfLeader(brokerInfo, partition),
```

Comment:
The follower should also be removed right?

Reply:
We are not keeping track of followers on the broker any more.

Reply:
ok thanks